py/specs/select.toml

```diff
@@ -0,0 +1,50 @@
+[setup]
+statement_files = ["sql/uservisits.sql"]
+statements = [
+    "copy uservisits from 's3://crate.amplab/data/1node/uservisits/part-*' with (compression = 'gzip')",
+    "refresh table uservisits"
+]
+
+
+[[queries]]
+# GET
+statement = "select * from uservisits where \"sourceIP\" = '25.193.131.52'"
+iterations = 100
+
+[[queries]]
+# MULTI_GET
+statement = """select * from uservisits where
+\"sourceIP\" = '162.122.2.75' or
```

Comment:
probably because you can write key-value pairs also like this:
`"key" = "value"`
https://github.com/toml-lang/toml#keyvalue-pair